packages/contracts-bedrock/interfaces/L1/IOPContractsManager.sol

```diff
@@ -166,6 +166,13 @@ interface IOPContractsManager {
     /// @param upgrader Address that initiated the upgrade
     event Upgraded(uint256 indexed l2ChainId, ISystemConfig indexed systemConfig, address indexed upgrader);
 
+    /// @notice Emitted when a new game type is added to a chain
+    /// @param l2ChainId Chain ID of the chain
+    /// @param gameType Type of the game being added
+    /// @param disputeGame Address of the deployed dispute game
+    /// @param delayedWETH Address of the WETH contract used by the game
+    event GameTypeAdded(uint256 indexed l2ChainId, GameType indexed gameType, IDisputeGame disputeGame, IDelayedWETH delayedWETH);
```

Comment:
should we index the game address? potentially useful for quickly finding what chain a give game belongs to
```suggestion
    event GameTypeAdded(uint256 indexed l2ChainId, GameType indexed gameType, IDisputeGame indexed disputeGame, IDelayedWETH delayedWETH);
```

Reply:
Resolving as the event has been updated based on proofs' input.